tox.ini

```diff
@@ -27,6 +27,8 @@ envlist =
 deps =
     coverage
     pytest
+extras =
+    extclick
```

Comment:
not sure if this is confusing, but i'd rather not call the extra the same thing as the package it installs. Open to name suggestions

Reply:
Maybe `cli`?

Reply:
If we're calling it ext should things go in ext instead of contrib?

Reply:
Oh and definitely not cli there's other cli frameworks out there.